packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/TileProviderController.java

```diff
@@ -66,37 +65,43 @@ Tile getTile() {
         return TileProvider.NO_TILE;
       }
       try {
-        return Convert.interpretTile(result);
+        if (result == null) {
```

Comment:
Opportunistic safety fix; the IDE was flagging that we weren't handling null `result` here.